docs/installing/manual_install.md

```diff
@@ -636,14 +642,28 @@ Create a directory for optional Alaveteli configuration
 Copy the example VirtualHost configuration file. You will need to change all
 occurrences of `www.example.com` to your URL
 
+For Wheezy and Precise:
```

Comment:
We've used bold for OS-specific instructions – I think we should continue to do so.